lib/talktome/version.rb

```diff
@@ -2,7 +2,7 @@ module Talktome
   module Version
     MAJOR = 0
     MINOR = 2
-    TINY  = 0
+    TINY  = 1
```

Comment:
Bump to 0.3.0. Upgrading path to >= 2.0 requires a minor change at least.

Reply:
Ok